src/main/java/com/example/java/test/junior/developer/service/UserService.java

```diff
@@ -21,7 +22,7 @@ public class UserService {
   private final KeycloakAdminClient keycloakAdminClient;
 
   @Transactional
-  public UserDto createUser(UserRequestDto userRequestDto) {
+  public UserDto createUser(@Valid UserRequestDto userRequestDto) {
```

Comment:
Remove it here. Use `@Valid` on controller layer.

Reply:
strange, but i have this code
  @Transactional
  public UserDto createUser(@Valid UserRequestDto userRequestDto) {
    var userKeycloak = userMapper.toUserRepresentation(userRequestDto);.....